.docker/Pipfile

```diff
@@ -23,35 +23,38 @@ librosa = ">=0.8.1"
 matplotlib = ">=3.4,<3.5.3"
 monai = "*"
 nbval = "*"
+nncf = {git = "https://github.com/openvinotoolkit/nncf.git"}
+numpy = ">=1.21.0"
 onnx = ">=1.11.0,<1.12.0"
+opencv-python = "*"
 openvino-dev = {extras = ["onnx","tensorflow2"], version = "==2022.3.0"}
 openvino-telemetry = "==2022.3.0"
 ovmsclient = "*"
 paddle2onnx = ">=0.6,<=0.9.6"
 paddleclas = ">=2.2"
 paddlenlp = ">=2.0.8"  # pin to avoid conflict with requests
-paddlepaddle = "==2.4.*"
+paddlepaddle = ">=2.4.0"
 Pillow = ">=8.3.2"
 ppgan = ">=2.1.0"
 psutil = "*"
 pyclipper = ">=1.2.1"
 pygments = ">=2.7.4"
-pytorch_lightning = "*"
+pytorch-lightning = "*"
 pytube = ">=12.1.0"
 rsa = ">=4.7"
 scikit-image = ">=0.19.2"
+scipy = "*"
 seaborn = ">=0.11.0"
 setuptools = ">56.0.0"
 shapely = ">=1.7.1"
 supervisor = "==4.1.0"
-tensorflow_datasets = "*"
+tensorflow = ">=2.5,<=2.9.3"
+tensorflow-datasets = "*"
```

Comment:
please pin tensorflow-datasets to 4.2.0 for docker, there is some issues with downloading dataset in 305 notebook for latest versionh

Reply:
or probably we should update tensorflow-datasets version in requirements.txt and use `imagenette/160px-v2`, imagenette/160px was removed in newer versions, that is why it can not be downloaded
https://www.tensorflow.org/datasets/catalog/imagenette

Reply:
I decided to go with 4.2.0 now to mitigate the risk of incompatible dependencies. But I agree it should be updated in the future.